test/test_files/abl_godunov_rayleigh_damping/abl_godunov_rauleigh_damping.inp

```diff
@@ -0,0 +1,77 @@
+#¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨#
+#            SIMULATION STOP            #
+#.......................................#
+time.stop_time               =   22000.0     # Max (simulated) time to evolve
+time.max_step                =   10          # Max number of time steps
+
+#¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨#
+#         TIME STEP COMPUTATION         #
+#.......................................#
+time.fixed_dt         =   0.5        # Use this constant dt if > 0
+time.cfl              =   0.95         # CFL factor
+
+#¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨#
+#            INPUT AND OUTPUT           #
+#.......................................#
+time.plot_interval            =  10       # Steps between plot files
+time.checkpoint_interval      =  5       # Steps between checkpoint files
+
+#¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨#
+#               PHYSICS                 #
+#.......................................#
+incflo.gravity          =   0.  0. -9.81  # Gravitational force (3D)
+incflo.density             = 1.0          # Reference density 
+
+incflo.use_godunov = 1
+incflo.diffusion_type = 2
+transport.viscosity = 1.0e-5
+transport.laminar_prandtl = 0.7
+transport.turbulent_prandtl = 0.3333
+turbulence.model = Smagorinsky
+Smagorinsky_coeffs.Cs = 0.135
+
+
+incflo.physics = ABL
+ICNS.source_terms = BoussinesqBuoyancy CoriolisForcing ABLForcing RayleighDamping
+BoussinesqBuoyancy.reference_temperature = 300.0
+ABL.reference_temperature = 300.0
+CoriolisForcing.latitude = 41.3
+ABLForcing.abl_forcing_height = 90
+
+RayleighDamping.time_scale=40. #in seconds
+RayleighDamping.damping_length=1000 #in metres
+RayleighDamping.full_damping_length = 500 #in metres
+RayleighDamping.reference_velocity = 15. 0.0 0.0 #in metres per second
+
+incflo.velocity = 6.128355544951824  5.142300877492314 0.0
+
+ABL.temperature_heights = 650.0 750.0 1000.0
+ABL.temperature_values = 300.0 308.0 308.75
```

Comment:
Also, I had a couple general questions -- the test case right now has a Rayleigh damping zone which overlaps with the inversion layer.  For the purposes of a test, should we remove the inversion layer so that we know that any sponging of disturbances is due to the the rayleigh damping alone?  

Related to that, have we established whether the Rayleigh damping zone should generally start above or below the inversion layer height?

Reply:
I would think that the Rayleigh damping has to be at the top, because it is focused on the behavior at the boundary. How would you like me to change the input file to remove the inversion layer? What should be removed / replaced?

Reply:
Thanks @mbkuhn.  In the regression test input file, I would use a flat temperature, like this:
```
ABL.temperature_heights = 650.0 750.0 1000.0
ABL.temperature_values = 300.0 300.0 300.0
```
And that way we can be sure that any velocity manipulation is due to the Rayleigh damping itself and not due to any possible buoyancy effects.  In the real case we could have both, and just put the Rayleigh damping zone above the inversion layer near the top boundary.